scripts/west_commands/runners/nrfjprog.py

```diff
@@ -8,9 +8,17 @@
 import os
 import shlex
 import sys
+from intelhex import IntelHex
```

Comment:
Needs to be wrapped in a try; please see https://github.com/zephyrproject-rtos/zephyr/blob/master/scripts/west_commands/runners/core.py#L318 for the policy and here for examples:

https://github.com/zephyrproject-rtos/zephyr/blob/master/scripts/west_commands/runners/openocd.py#L9
https://github.com/zephyrproject-rtos/zephyr/blob/master/scripts/west_commands/runners/openocd.py#L113